assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -79,6 +81,9 @@ function SingleSelectionMovable( {
 		}
 	};
 
+	const isTextElement = 'text' === selectedElement.type;
+	const shouldAdjustFontSize = isTextElement && selectedElement.content.length && keepRatioMode;
```

Comment:
I am a bit confused about `keepRatio` / `keepRatioMode`. Text elements have a "Keep ratio" checkbox, yet `SingleSelectionMovable` only supports `keepRatio` for image elements. Should this be changed? If not, the `keepRatioMode` check here is irrelevant as ratio is never kept for text elements.

Reply:
`keepRatio` checkbox is unrelated to keeping ratio when resizing, `keepRatio` in the sidebar is for manually changing the width and height only.

`keepRatioMode` basically means that resizing is happening from corners, I'll just rename this for clarity to `resizingFromCorners` or sth like that.